test/test_regset.c

```diff
@@ -382,6 +382,8 @@ static char* p7[] = {
   "0+", "1+", "2+", "3+", "4+", "5+", "6+", "7+", "8+", "9+",
 };
 
+static char* p8[] = {".*", "a"};
```

Comment:
I was having a difficult time following the test suite, oddly if I switch the position of the two regexes here the test fails 🤔

Reply:
Thank you.
It was an another bug for empty string search.

Reply:
ah! awesome -- thanks for the quick fix :)